src/components/modebar/modebar.js

```diff
@@ -88,7 +88,9 @@ proto.update = function(graphInfo, buttons) {
             }
 
             if(fullLayout.modebar.orientation === 'v') {
-                this.element.prepend(logoGroup);
+                d3.select(this.element).insert(function() {
```

Comment:
Hmm. Does

```js
d3.select(this.element).insert(logoGroup, ':first-child');
```

work also?

Reply:
I tried that initially but it does not work:
![screenshot_2019-01-28_17-07-49](https://user-images.githubusercontent.com/301546/51869674-47f97580-231f-11e9-923a-3c4b45eface3.png)

Reply:
Ha right, does

```js
d3.select(this.element).insert(d3.select(logoGroup), ':first-child');
```

work?

Reply:
> d3.select(this.element).insert(d3.select(logoGroup), ':first-child');

Also fails with: 
![screenshot_2019-01-28_17-18-23](https://user-images.githubusercontent.com/301546/51870142-bd197a80-2320-11e9-83a5-3831ced4ead1.png)

Would you prefer that I to turn the snippet from this PR into a one-liner?

Reply:
Cool. Thanks for checking!

Reply:
Non- 🚫, but at that point the vanilla solution seems simpler:

https://github.com/plotly/plotly.js/blob/ea7941b97fbaf11967f3c118a8e130036c8169f4/src/plot_api/subroutines.js#L148

https://github.com/plotly/plotly.js/blob/ea7941b97fbaf11967f3c118a8e130036c8169f4/src/lib/svg_text_utils.js#L105-L106

Reply:
Thanks @alexcjohnson! I agree it is simpler. Done in commit 4b164a26506e60f29e854600c76042cc9b94c53d

Updated demo https://plnkr.co/edit/3Rfo4svtr6PUDCiCRM3l?p=preview